### PR TITLE
[YSTORM-1661] Added acl validation config 1.x Branch PR

### PR DESCRIPTION
**Pull request by redsanket**

### CODE REVIEW

File: storm-core/src/jvm/org/apache/storm/blobstore/BlobStoreAclHandler.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
     public static final List<AccessControl> DEFAULT = new ArrayList<AccessControl>();
     private Set<String> _supervisors;
     private Set<String> _admins;
+    private boolean doAclValidation;
```
Comment (knusbaum) [inline storm-core/src/jvm/org/apache/storm/blobstore/BlobStoreAclHandler.java]: This should probably be final.

**Comment by HeartSaVioR** [comment]
+1

**Comment by HeartSaVioR** [comment]
Sorry I didn't indicate the compile error from the CI. Revoking +1.
@redsanket Could you check the compile issue? doAclValidation should be initialized when it's declared as final but there's a path to not initialized in constructor.

**Comment by redsanket** [comment]
@HeartSaVioR reverted the last commit, hopefully it is good to go

**Comment by HeartSaVioR** [comment]
@redsanket I cherry-picked to merge this to 1.x-branch, so this can be closed.

**Comment by knusbaum** [comment]
This was cherry picked. Closing.

**Pull request closed without merging**